tests/units/StringTest.cpp

```diff
@@ -3,7 +3,7 @@
 
 TEST(StringTest, makeInt32)
 {
-    char *test = "52";
+    const char *test = "52";
     int32_t result_int32 = 52;
     EXPECT_EQ(String::makeInt32(test), result_int32);
     EXPECT_EQ(typeid(String::makeInt32(test)), typeid(result_int32));
```

Comment:
No need to test `typeid`.
Avoid using `typeid`, `dynamic_cast` and `exception` whenever it is possible. Only if there is a real need.